test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -4195,6 +4196,20 @@ TEST_F(AtenXlaTensorTest, TestRandperm) {
                           cpp_test::GetIgnoredCounters());
 }
 
+TEST_F(AtenXlaTensorTest, TestRandpermOut) {
```

Comment:
does this test run as expected locally? here is how you can test it locally:

```
$ test/cpp/build/test_ptxla --gtest_filter=AtenXlaTensorTest.TestRandpermOut
```

Reply:
yeah, this was what I used to run the test.